index.js

```diff
@@ -81,21 +81,8 @@ class WebTorrent extends EventEmitter {
       this.peerId, this.nodeId, this.torrentPort
     )
 
-    if (this.tracker) {
-      if (typeof this.tracker !== 'object') this.tracker = {}
```

Comment:
I think you missed this line

Reply:
no? I just simplified it to 1 if statement

Reply:
sure did